src/CommunityToolkit/Xamarin.CommunityToolkit/Helpers/FormsToolkit.ios.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UIKit;
+
+namespace Xamarin.CommunityToolkit.Helpers
+{
+	public static class FormsToolkit
```

Comment:
Good idea, this should be internal as we don't want to support the concept of `FormsToolkit`. This might get completely removed depending on the decision made in https://github.com/xamarin/XamarinCommunityToolkit/pull/653#discussion_r540208852

Reply:
This code has been removed since Xamarin.Forms 5+ support iOS 9+. That makes the only API in this class obsolete.

Resolving conversation